docs/user-guide.md

```diff
@@ -36,14 +36,18 @@ pipelines:
       - path: /banking/testing
         regions: eu-central-1
 
-  - name: vpc
+  # Github source example
+  - name: vpc-example # Pipeline name MUST be equal to Github repository name.
     type: github-cloudformation
     action: replace_on_failure
     params:
-      - SourceAccountId: 8888877777777
+      - ProjectName: "vpc-example" # Name of the Github repository
```

Comment:
This parameter is not mandatory, ADF itself generates the value of this parameter and passes it in at Pipeline creation time, it will just uses the project name *(aka the pipeline name)*.